ceph_iscsi_config/client.py

```diff
@@ -663,6 +663,8 @@ def manage(self, rqst_type, committer=None):
 
                         # persist the config update
                         config_object.commit()
+                        if config_object.error:
+                            self.error = True
```

Comment:
You should set the error_msg here and below.

Reply:
Yes, you are right, i will increase it.